src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -659,4 +659,10 @@ public int getLastContactInHours()
     {
         return lastContactInHours;
     }
+
+    @Override
+    public int getRaidLevel()
```

Comment:
Irrelevant, javadoc is on the Interface. That is enough, no need to blow up files with the same javadoc twice.